lib/client/jsx/components/folder-view.jsx

```diff
@@ -64,21 +74,34 @@ class FolderView extends React.Component {
 
     let buttons = [
       { onClick: this.selectFolder.bind(this), title: 'Create folder', icon: 'folder', overlay: 'plus', role: 'editor' },
-      { onClick: this.selectFile.bind(this), title: 'Upload file', icon: 'upload', role: 'editor' }
+      { onClick: this.selectFile.bind(this), title: 'Upload file', icon: 'upload', role: 'editor' },
+      { onClick: this.selectDir.bind(this), title: 'Upload Directory', icon: 'cloud-upload-alt', role: 'editor' },
     ];
```

Comment:
This seems dumb, but I think the cloud upload icon is confusing and doesn't suggest folder upload, and conflicts with the way we are using the same icon to mean "link to metis" on Timur - but there doesn't seem to be a useful folder upload icon in Font Awesome. Might we instead have the 'upload' button drop a menu from which we could select "Upload Files" or "Upload Folder" to spawn the appropriate dialog? Ruins this control bar's behavior...